### DISCUSSION TITLE
登录用户信息不需要存储与数据库吗？

### DISCUSSION THREAD

**Discussion by tian-yan** [wechatpy/wechatpy#695] [Q&A] [answered]
在代码里面，没有看到`models.py`文件，如果微信公众号菜单直接指向网页，微信用户点击该菜单进入网页时，网页授权后，用户信息如何存储到网站的数据库？

如果用户信息不存储到数据库的话，用户如何在点击公众号菜单进入网页时，如果在该网页上发表评论？（评论内容都是存储在网站的数据库里面，有一个`foreign_key`指向用户的。

谢谢。

**Comment by Athrun59** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by JoshYuJump** [comment, marked as answer]
存储到数据库应该是你自己的应用实现的
